### PR TITLE
DFA-2050 DFA-2054 User entered the wrong security code too many times

### PR DESCRIPTION
**Pull request by BorisP-GDS**
User entered the wrong security code too many times - frontend only

### CODE REVIEW

**Review by HarshVasudev — APPROVED** [review]

**Review by CharlesIC — CHANGES_REQUESTED** [review]
The base template already includes `- GOV.UK Sign In` in the page title

File: express/src/routes/testing-routes.ts [hunk 1]
```diff
@@ -492,4 +492,35 @@ router.post("/account-exists", passwordValidator("create-account/account-exists.
...
+// When implementing the backend, depending on journey we should request new code and after that redirect (/create/enter-mobile or /sign-in-otp-mobile) or render the appropriate template - depending on implementation
+router.post("/wrong-text-code", async (req, res) => {
+    res.render("check-mobile.njk");
+});
+
+//// Testing routes for 'You entered the wrong email security code too many times' page
+router.get("/wrong-email-code", (req, res) => {
+    res.render("wrong-otp-too-many-times.njk", {
+        values: {
+            wrongOtpPageTitle: "Wrong email code entered too many times - GOV.UK Sign In",
```
Comment (CharlesIC) [inline express/src/routes/testing-routes.ts:516]:
```suggestion
            pageTitle: "Wrong email code entered too many times",
```

File: express/src/views/wrong-otp-too-many-times.njk [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{% extends "layout.njk" %}
+
+{% set pageTitle = values.wrongOtpPageTitle %}
```
[the remark this answers is not in the document]
Comment (BorisP-GDS) [reply express/src/views/wrong-otp-too-many-times.njk]: Updated

File: express/src/routes/testing-routes.ts [hunk 3]
```diff
@@ -492,4 +492,35 @@ router.post("/account-exists", passwordValidator("create-account/account-exists.
     res.render("check-mobile.njk");
 });
 
+//// Testing routes for 'You entered the wrong mobile security code too many times' page
+//// The code below is for frontend/visual testing purposes only
+
+router.get("/wrong-text-code", (req, res) => {
+    res.render("wrong-otp-too-many-times.njk", {
+        values: {
+            wrongOtpPageTitle: "Wrong text message code entered too many times - GOV.UK Sign In",
```
[the remark this answers is not in the document]
Comment (BorisP-GDS) [reply express/src/routes/testing-routes.ts]: updated

File: express/src/routes/testing-routes.ts (hunk 1 above)
Comment (BorisP-GDS) [reply express/src/routes/testing-routes.ts]: Updated

**Review by CharlesIC — APPROVED** [review]

**Pull request merged**